app/controllers/preferences_controller.rb

```diff
@@ -24,6 +24,16 @@ def create
     end
   end
 
+  def destroy
+    @preference = Preference.find(params[:id])
+
+    if @preference.destroy
+      redirect_to preferences_path, notice: t('views.preferences.destroy_success')
+    else
+      redirect_to preferences_path, alert: t('views.preferences.destroy_failure'), status: :unprocessable_entity
+    end
+  end
```

Comment:
Same as other PRs, it would be great if you could do some feature test for the visual side of the functionality